bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/categorylist/FeaturedCategoryListImpl.java

```diff
@@ -73,6 +74,7 @@
     private Map<String, Asset> assetOverride;
     private Page categoryPage;
     private AbstractCategoriesRetriever categoriesRetriever;
+    private List<CategoryTree> categories;
```

Comment:
You don't need this. The `retriever` classes will automatically populate and store the result once you call the `fetchCategories` method.